tratto/src/main/java/star/tratto/util/javaparser/TypeUtils.java

```diff
@@ -0,0 +1,295 @@
+     * primitive types.
+     *
+     * @param classGetNameComponent a ClassGetName type
+     * @return true iff the ClassGetName component is a field descriptor
+     * @see TypeUtils#primitiveFieldDescriptors
+     */
+    private static boolean isPrimitiveFieldDescriptor(String classGetNameComponent) {
+        return primitiveFieldDescriptors.contains(classGetNameComponent);
+    }
+
+    /**
+     * Gets the ClassGetName form of the base component type for a given
+     * ClassGetName array type. For example,
+     *     "[[I"    =>    "I"
+     *     "[Ljava.lang.String;"    =>    "java.lang.String"
+     *
+     * @param classGetNameArray a ClassGetName form of an array type
+     * @return the ClassGetName form of the component type without arrays
+     */
+    private static String classGetNameComponentType(String classGetNameArray) {
+        int arrayLevel = getArrayLevel(classGetNameArray);
+        if (arrayLevel == 0) {
+            return classGetNameArray;
+        }
+        String classGetNameComponent = classGetNameArray.replaceAll("\\[", "");
+        if (isPrimitiveFieldDescriptor(classGetNameComponent)) {
+            return classGetNameComponent;
+        } else {
+            classGetNameComponent = classGetNameComponent.replaceAll(";", "");
+            // for arrays of Objects, remove "L" prefix
+            return classGetNameComponent.substring(1);
+        }
+    }
+
+    /**
+     * Converts a ClassGetName form of a type to its corresponding
+     * ClassGetSimpleName form.
+     *
+     * @param classGetName a ClassGetName form of a type
+     * @return the corresponding ClassGetSimpleName form of a type
+     */
+    private static String classGetNameToClassGetSimpleName(
+            String classGetName
+    ) {
+        int arrayLevel = getArrayLevel(classGetName);
+        classGetName = classGetNameComponentType(classGetName);
+        if (isPrimitiveFieldDescriptor(classGetName)) {
+            // ClassGetName uses field descriptors for arrays of primitive types
+            classGetName = Signatures.fieldDescriptorToBinaryName(classGetName);
+        } else {
+            classGetName = getInnermostClassNameFromClassGetName(classGetName);
+        }
+        return addArrayLevel(classGetName, arrayLevel);
+    }
+
+    /**
+     * Converts a list of ClassGetName forms of types to their corresponding
+     * ClassGetSimpleName forms.
+     *
+     * @param classGetNames ClassGetName forms of types
+     * @return the corresponding ClassGetSimpleName forms of types
+     */
+    public static List<String> classGetNameToClassGetSimpleName(
+            List<String> classGetNames
+    ) {
+        return classGetNames
+                .stream()
+                .map(TypeUtils::classGetNameToClassGetSimpleName)
+                .collect(Collectors.toList());
+    }
+
+    /**
+     * Removes type arguments and semicolons from a parameterized type name.
```

Comment:
The argument is not a parameterized type name.  Is it a variable definition?  Please explain what it is.